gramps/gen/lib/serialize.py

```diff
@@ -71,3 +73,91 @@ def from_json(data):
     :rtype: object
     """
     return json.loads(data, object_hook=__object_hook)
+
+
+def to_struct(obj):
+    """
+    Convert a Gramps object into a struct.
+
+    :param obj: The object to be serialized.
+    :type obj: object
+    :returns: A dictionary.
+    :rtype: dict
+    """
+    return json.loads(to_json(obj))
+
+
+def from_struct(struct):
+    """
+    Convert a struct into a Gramps object.
+
+    :param data: The dictionary to be unserialized.
```

Comment:
Should this be "struct" rather than "data"?

Reply:
Do you know about github suggestions (box in edit box with +/-)? Then I could have accepted it :)

Reply:
Thanks!